### DISCUSSION TITLE
How to add list of scheduled and storage event based triggers inside my ADF, to CSV file?

### DISCUSSION THREAD

**Discussion by gajulakumar** [Azure-Player/azure.datafactory.tools#228] [Q&A]
I have a list of scheduled and storage even based triggers with different time stamp how can i fit all this triggers into CSV file

Getting below error 
ADFT0010: Wrong path defined in config for object(path): trigger.trigger_*(properties.typeProperties.*)

**Comment by NowinskiK** [comment]
Can you show what exactly did you try?
